app/models/concerns/devise_setup.rb

```diff
@@ -0,0 +1,29 @@
+module DeviseSetup
+  require 'hours'
+
+  extend ActiveSupport::Concern
+
+  included do
+    if ActiveRecord::Base.connection.table_exists? 'users'
+      if User.count == 0
+        devise :database_authenticatable,
+               :recoverable,
+               :rememberable,
+               :trackable,
+               :validatable,
+               :confirmable,
+               :registerable,
+               :invitable
+      else
+        devise :database_authenticatable,
+               :recoverable,
+               :rememberable,
+               :trackable,
+               :validatable,
+               :confirmable,
+               :invitable
+      end
```

Comment:
Yeah makes sense. Let's just add a rake task that lets you set up the first user?